src/wmnds/components/lists/unordered-list/_unordered-list.scss

```
@@ -0,0 +1,18 @@
+.wmnds-unordered-list {
+  list-style-position: inside;
+  padding-inline-start: 0;
+
+  ul {
+    padding-inline-start: 2 * $size-sm;
```

```suggestion
    padding-inline-start: $size-lg;
```

Same as previous comment with sizing.